ethereum/permissioning/src/test/java/tech/pegasys/pantheon/ethereum/permissioning/NodeLocalConfigPermissioningControllerTest.java

```diff
@@ -194,19 +194,21 @@ public void whenNodesListeningPortsAreDifferentItShouldNotBePermitted() {
   }
 
   @Test
-  public void whenCheckingIfNodeIsPermittedDiscoveryPortShouldNotBeConsideredIfAbsent() {
+  public void
+      whenCheckingIfNodeIsPermittedDiscoveryPortShouldBeConsidered_implicitDiscPortMismatch() {
```

Comment:
@lucassaldanha - How do you think these permissions should work wrt discovery?  I've changed `EnodeURL` so that it can represent the case where discovery is actually disabled.  So now the `discoveryPort` is only empty if discovery is disabled which meant I had to rework these checks a bit.  

I think the code would be simpler if we just ignore the discovery port and use `EnodeURL.sameListeningEndpoint` to check enodes against the whitelist.  Alternatively, we could check that the discovery port values are the same (so if one peer has discovery enabled and the other doesn't, they don't match). 
 What do you think?

Reply:
I actually just went ahead and updated `NodeLocalConfigPermissioningController` to ignore the discovery port.  Let me know if you disagree.